package-support.json

```diff
@@ -0,0 +1,21 @@
+{
+  "versions": [
+    {
+      "version": "*",
+      "target": {
+        "node": "lts"
```

Comment:
According to [`node` namespace](https://github.com/nodejs/package-maintenance/blob/main/docs/PACKAGE-SUPPORT.md#node-namespace), should this be `active`? Since we support node 10,12,14,15 (whereas `lts` is only 10,12)

Reply:
LTS is now 10,12,14, but regardless good point.

Reply:
@KevinEady fixed.